### DISCUSSION TITLE
播放倍率没有记忆功能了

### DISCUSSION THREAD

**Discussion by 415simple** [Richasy/Bili.Uwp#856] [问答] [answered]
在三月更新v2还有这个功能，安装好后点二倍速以后软件就记住了，昨天我更新到三月v3后这个功能就没有了，现在需要每次播放视频的时候手动调节到二倍速就有点繁琐了。上了论坛发现大家都没反应这问题，我也找不到解决方案，希望在下次版本更新时能加回记忆的功能，辛苦开发者大大了~

**Comment by Richasy** [comment, marked as answer]
#849 ，将在4月更新
